lms/djangoapps/grades/api/v0/views.py

```diff
@@ -0,0 +1,142 @@
+""" API v0 views. """
+import logging
+
+from django.http import Http404
+from edx_rest_framework_extensions.authentication import JwtAuthentication
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from rest_framework import status
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from lms.djangoapps.ccx.utils import prep_course_for_grading
+from lms.djangoapps.courseware import courses
+from lms.djangoapps.grades.new.course_grade import CourseGradeFactory
+from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
+
+log = logging.getLogger(__name__)
+
+
+class CurrentGradeView(GenericAPIView):
```

Comment:
Please rename to `UserGradeView` as this is specifically for users.  I believe "current" is implicit as we usually return the current value.